src/server/components/formRunner/helpers.ts

```diff
@@ -83,3 +94,51 @@ export function parseFormRunnerWebhookObject<T>({
     return acc;
   }, {}) as T;
 }
+
+export function getNewSessionWebhookData(listType: string, listItemId: string, questions: Array<Partial<FormRunnerQuestion>> | undefined, message: string): FormRunnerNewSessionData {
+  const protocol = isLocalHost ? "http" : "https";
+  const callbackUrl = `${protocol}://${SERVICE_DOMAIN}/ingest/${listType}/${listItemId}`;
+  const redirectPath = `/summary`;
+  const options = {
+    message,
+    callbackUrl,
+    redirectPath
+  };
+
+  const newSessionData: FormRunnerNewSessionData = {
+    questions,
+    options,
+    name: "Changes required"
+  };
+  return newSessionData;
+}
+
+
+export async function parseJsonFormData(listType: string, isUnderTest?: boolean): Promise<Array<Partial<FormRunnerQuestion>>> {
+
+  const formsJsonFile = (isUnderTest === true) ? `/forms-json/${listType}.json` : `../src/server/components/formRunner/forms-json/${listType}.json`;
+  const fileContents = await fs.promises.readFile(path.join(__dirname, formsJsonFile), "utf8");
```

Comment:
ah this is an interesting one.  Because it's trying to read the file rather than access the page, it needs the physical file location rather than the path relative to the server root.  I was playing around with options but couldn't find a nice compromise, including the method you suggested.  Any other ideas to make this a bit more succinct?